src/app/clients/clients-view/custom-dialogs/upload-image-dialog/upload-image-dialog.component.html

```diff
@@ -2,11 +2,11 @@ <h1 mat-dialog-title>Upload Client Image</h1>
 
 <div>
 
-  <mifosx-file-upload flex="60%" (change)="onFileSelect($event)"></mifosx-file-upload>
+  <mifosx-file-upload flex="60%" (change)="onFileSelect($event)" acceptFilter=".jpg,.jpeg,.gif,.tif,.tiff,.png|image/*"></mifosx-file-upload>
```

Comment:
Looks incorrect to me.
Based on the default filter on the backend the following are whitelisted: .jpg,.jpeg,.png.